support-frontend/assets/helpers/forms/stripe.ts

```diff
@@ -1,50 +1,69 @@
+import type { ActiveProductKey } from '@guardian/support-service-lambdas/modules/product-catalog/src/productCatalog';
 import type { Stripe as StripeJs } from '@stripe/stripe-js';
 import { loadStripe } from '@stripe/stripe-js/pure';
 import { useEffect, useState } from 'react';
 import type { ContributionType } from 'helpers/contributions';
 import type { IsoCountry } from 'helpers/internationalisation/country';
 import type { IsoCurrency } from '../internationalisation/currency';
 
-export type StripeAccount = 'ONE_OFF' | 'REGULAR';
+export type StripeAccountType = 'ONE_OFF' | 'REGULAR';
```

Comment:
I just renamed this to make it clearer what it represents. Quite a few of the changes in this PR are in files which reference this type.